src/Twig/AppExtension.php

```diff
@@ -5,10 +5,12 @@
 use Caxy\HtmlDiff\HtmlDiff;
 use DateTime;
 use Doctrine\Bundle\DoctrineBundle\Registry;
-use Elasticsearch\Client;
+use Elastica\ResultSet;
```

Comment:
Don't we need to use the common result set? 

https://github.com/ems-project/EMSCommonBundle/blob/develop/src/Elasticsearch/Elastica/ResultSet.php

Reply:
It's an hook to make the client 5 working with es7

Reply:
okay aggree